bump.js

```diff
@@ -47,7 +47,8 @@ module.exports = async function ({ context, inputs }) {
   const branchName = `release/${newVersion}`
 
   await run('git', ['checkout', '-b', branchName])
-  await run('git', ['commit', '-am', newVersion])
+  await run('git', ['add', '-A'])
+  await run('git', ['commit', '-m', newVersion])
```

Comment:
oh by the way, we need to make this customizable. if we don't have an issue, please open one. anything where we're making decisions we need to allow the user to customize the decision. in this case, we may use a formatting utility so that the user can provide a commit message containing a placeholder that we replace with the version